Source/RN4UE4/Replica.h

```diff
@@ -12,7 +12,7 @@
 #include "VariableDeltaSerializer.h"
 #include "GetTime.h"
 #include "Rand.h"
-
+#include "Materials/Material.h"
```

Comment:
Why is this needed?

Reply:
It gives an error that UMaterial wasn't found if I don't put the library

Reply:
What is  new that needs it?